orion/pkg/repos/achieve_repo.go

```diff
@@ -185,6 +221,22 @@ func (ar *achieveRepo) Delete(id uint) error {
 	return handleSqlExecResult(execResult, 1, "achievement was not deleted")
 }
 
+func (ar *achieveRepo) Publish(id uint) error {
+	statement := "UPDATE achievements SET updated_at=? WHERE id=?"
```

Comment:
should be editing `updated_at` and `published_at`

Reply:
I changed this already